### ISSUE TITLE
[FEATURE] Possibility to @push to content section

### ISSUE THREAD

**Issue by AMoktar** [jeroennoten/Laravel-AdminLTE#1045]
https://github.com/jeroennoten/Laravel-AdminLTE/blob/2382d0623d33bb37c946c70e4c940371b8b24c25/resources/views/partials/cwrapper/cwrapper-default.blade.php#L22-L25

please add this stack 
```
    <div class="content">
        <div class="{{ config('adminlte.classes_content') ?: $def_container_class }}">
            @stack('content') // <<<<<----
            @yield('content')
        </div>
```

as https://github.com/jeroennoten/Laravel-AdminLTE/blob/2382d0623d33bb37c946c70e4c940371b8b24c25/resources/views/page.blade.php#L5-L8

so it is easy to create a parent layout like 
```
@extends('adminlte::page')




@push('content')
<div class="container">
    <div class="row justify-content-center">
        <div class="col-md-10">
            @if (session()->has('message'))
            <div class="alert alert-info" role="alert">
                <p>{{ session()->get('message') }}</p>
            </div>
            @endif
        </div>
    </div>
</div>
@endpush



@push('css')
<link href="{{ assets('css/custom.admin.css') }}" rel="stylesheet">
@endpush
```

currently on release 3.7 the css above is working , but the content push doesnot 

Thanks

**Comment by Shidersz** [comment]
What you want to do, may actually be done if you yield a new section, for example:

**my-app-layout.blade.php**
```blade
@extends('adminlte::page')

@section('content')
<div class="container">
    <div class="row justify-content-center">
        <div class="col-md-10">
            @if (session()->has('message'))
            <div class="alert alert-info" role="alert">
                <p>{{ session()->get('message') }}</p>
            </div>
            @endif
        </div>
    </div>
</div>
@stack('my_layout_content')
@yield('my_layout_content')
@endsection

@push('css')
  <link href="{{ assets('css/custom.admin.css') }}" rel="stylesheet">
@endpush
```

Then you will usually `@extends('my-app-layout')` and use the new `my_layout_content` section.

**Comment by AMoktar** [comment]
thanks man, my solution was sweet :))
Thanks

**Comment by Shidersz** [comment]
Ok, we will take this into consideration for a future release...

**Comment by AMoktar** [comment]
Thanks @Shidersz , You are awesome .

**Comment by dfsmania** [comment]
Closed by #1235
